### PR TITLE
Added cancellation token support in characteristic start and stop upd…

### PR DESCRIPTION
**Pull request by UwpDev**
Added cancellation token in Characteristic's StartUpdatesAsync and StopUpdatesAsync

### CODE REVIEW

File: Source/Plugin.BLE.Abstractions/CharacteristicBase.cs [hunk 1]
```diff
@@ -102,18 +102,18 @@ private CharacteristicWriteType GetWriteType()
...
                 throw new InvalidOperationException("Characteristic does not support update.");
             }
 
             Trace.Message("Characteristic.StartUpdates");
-            return StartUpdatesNativeAsync();
+            return StartUpdatesNativeAsync(cancellationToken);
         }
 
-        public Task StopUpdatesAsync()
+        public Task StopUpdatesAsync(CancellationToken cancellationToken = default)
```
Comment (janusw) [inline Source/Plugin.BLE.Abstractions/CharacteristicBase.cs:116]: Generally LGTM, but here the token is not being passed on to `StopUpdatesNativeAsync`!

Comment (janusw) [reply Source/Plugin.BLE.Abstractions/CharacteristicBase.cs:116]: Fixed in b917bf2e57c415d02b73f3f8f10ed07902d0a079.

**Review by janusw — APPROVED** [review]

**Comment by janusw** [comment]
Thanks for the contribution (and sorry for the late review)! Merged.

**Comment by UwpDev** [comment]
@janusw Thanks for reviewing and resolving the comment as well :)

**Pull request merged**